src/main/scala/akka/io/AsyncDnsResolver.scala

```diff
@@ -154,4 +183,26 @@ object AsyncDnsResolver {
 
   private[io] def isSrv(name: String) =
     name.nonEmpty && name(0) == '_'
+
+  private[io] val NameserverLine = """^\s*nameserver\s+(.*)$""".r
+
+  def parseSystemNameServers(resolvConf: File): Option[immutable.Seq[InetSocketAddress]] =
+    try {
+      val addresses =
+        for {
+          line <- Source.fromFile(resolvConf).getLines()
```

Comment:
ok, then I believe it is best left as-is - I shall introduce a comment to this extent.